README.adoc

```diff
@@ -1,48 +1,166 @@
+
+=== Custom Protocol Handler
+Starfix includes a custom URL Handler to handle `ide://` on the users machine.We have achieved this  by executing Registry in Windows and by .desktop file in Linux.Whenever a URI with scheme `ide://` is encountered Starfix app is launched with the arguement as the  URI.
+
+=== Browser Extension
+To be able to add a "Open In IDE" button on Web Pages and launching the protocol handler we need a browser extension .The extension also acts as the __integration between web and desktop__ . It swritten with  __plain HTML/CSS/JS__.
+
+
+== Supported Platforms:
 - Linux
 - Windows
 - Mac OS X
 
-## Quick Command reference:
-- Starting in Development mode : `mvn quarkus:dev`
-- Passing arguements: When you run with `mvn quarkus:dev` you can add `-Dquarkus.args=yourvalue` as arguments you want to pass into the command line.
+== Installation
+=== LINUX
+- Download the `starfix-linux.zip` file from the Latest Release.
+- Extract and Navigate to `installer` directory
+- On the terminal execute the following(ensure the current directory is `installer` ) 
+
+[source, bash]
+----
+./install
+----
+
+- Once you execute the above command you'll  be prompted for password (for sudo )
+- Test your installation by going to terminal and typing `starfix`
+- To set-up the Browser Extension navigate to this section: https://github.com/fahad-israr/starfish/blob/doc/README.adoc#browser-extension[Broswer Extension]
+
+=== WINDOWS
+- Download the `starfix-win64.zip` file from the Latest Realease
+- Extract and Navigate to `installer` directory
+- Now execute the file `install.cmd` (You can simply double click)
+- As the app is unrecognized, Windows may launch up a new Dialog titled as `Windows protected your PC` with description `Microsoft Defender SmartScreen prevented an unrecognised app from starting. Running this app might put your PC at risk.`
+- You may click on `More info` and the choose `Run anyway`
+- You will then be **prompted** with a series of **dialog boxes** asking **permission** for the operatons(modifying the registry). Please **Select Yes** in all those prompts to allow the installation.
+- Test your installation by typing `starfix` on the Terminal(Command Prompt or Windows PowerShell)
+- To set-up the Browser Extension navigate to this section: https://github.com/fahad-israr/starfish/blob/doc/README.adoc#browser-extension[Broswer Extension]
+
+=== MACOS
+- Download the `starfix-macos.zip` file from the Latest Realease
+- Extract and Navigate to `installer` directory
+- On the terminal execute the following(ensure the current directory is `installer` ) 
+
+[source, bash]
+----
+./install
+----
+
+- Once you execute the above command you'll  be prompted for password (for sudo )
+- Test your installation by going to terminal and typing `starfix`
+- To set-up the Browser Extension navigate to this section: https://github.com/fahad-israr/starfish/blob/doc/README.adoc#browser-extension[Broswer Extension]
```

Comment:
should not point to your personal repo

Reply:
resolved @https://github.com/starfixdev/starfix/pull/29/commits/102717f142cdff59a6a9ce92971c3e971c34342e